shared/src/commonMain/kotlin/com/bunbeauty/shared/presentation/createorder/CreateOrderViewModel.kt

```diff
@@ -450,11 +496,35 @@ class CreateOrderViewModel(
                         "$oldFinalCost $RUBLE_CURRENCY"
                     },
                     newFinalCost = "${cartTotal.newFinalCost} $RUBLE_CURRENCY",
+                    newFinalCostValue = cartTotal.newFinalCost,
                 )
             )
         }
     }
 
+    private fun getExtendedComment(
+        state: CreateOrder.DataState,
+        withoutChange: String,
+        changeFrom: String,
+    ): String {
+        return buildString {
+            state.comment.takeIf { comment ->
+                comment.isNotBlank()
+            }?.let { comment ->
+                append("$comment ")
```

Comment:
extract " " in paymentByCash
for example append(" ") or append(" (")

Reply:
add trim